.github/workflows/codeql.yml

```
@@ -0,0 +1,35 @@
+name: "CodeQL"
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+  schedule:
+  - cron: '54 3 * * 2'
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
```

Sounds like this doesn't apply here, can we omit them?

```suggestion
      # only required for workflows in private repositories
      actions: read
      contents: read
```

---

sure, done in https://github.com/IBM/cloudant-go-sdk/pull/444/commits/09c5773a17cd84d8b7d9fb63e74953ca2096f2c7